src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbolBase.cs

```diff
@@ -348,6 +384,37 @@ private void EnsureSignature()
             }
         }
 
+        // PROTOTYPE(semi-auto-props): Consider renaming to EnsureBackingFieldIsSynthesized.
+        // PROTOTYPE(semi-auto-props): Consider not using CompletionPart for this purpose.
```

Comment:
>Consider not using CompletionPart for this purpose. [](http://example.com/codeflow?start=39&length=51)

One of the goals that I would like us to achieve in the initial scoped down PR is that be able to compile basic scenarios like
```
public class Point
{
    public int X { get { return field; } set { field = value; } }
    public int Y { get { return field; } set { field = value; } }
}
``` 

without triggering extra binding of accessors. Basically, I would like us to record the fact that we tried binding accessors here through ```CSharpCompilation.TestOnlyCompilationData``` and assert in tests that it didn't happen. We will not be able to do that if we stick with completion. In fact, that guarantees that we will be doing extra binding.

Reply:
Done